pkg/checks/external/main.go

```diff
@@ -119,6 +123,7 @@ func New(client *kubernetes.Clientset, checkConfig *khcheckcrd.KuberhealthyCheck
 		OriginalPodSpec:          checkConfig.Spec.PodSpec,
 		PodSpec:                  checkConfig.Spec.PodSpec,
 		KubeClient:               client,
+		crdUID:                   checkConfig.GetObjectMeta().GetUID(),
```

Comment:
seems like instead of the khcheck crdUID, you want to reference the kuberhealthy deployment UID? im not sure we have something in the code that gets the kuberhealthy deployment UID... @integrii ? 

may need to add a function somewhere that finds the kuberhealthy deployment and gets the UID from it?

Reply:
I would think that the external checker pods that get spun up would have an owner reference that points to the kuberhealthy pod that spawned them (not the deployment or khcheck).  This way, if the kuberhealthy instance that is orchestrating the check disappears for some reason, the checker pods it was orchestrating (and had stateful data for) are also removed.

I think this means we can probably drop all the `crdUID` property stuff and just simply put in the hostname of the kuberhealthy instance that spawns the pod in the owner reference.

Reply:
So I will finally take a third stab at this and I will do as you say just point back at the kuberhealthy pod as the owner. I hopefully have something to push later today.